File: BackEnd/ssacle/src/main/java/S12P11D110/ssacle/domain/study/dto/StudyDTO.java
package S12P11D110.ssacle.domain.study.dto;

import S12P11D110.ssacle.domain.study.entity.Study;
import lombok.Builder;
import lombok.Data;

import java.util.List;
import java.util.Set;

@Data
@Builder
//스터디 정보를 담은 DTO
public class StudyDTO {
    private String studyId;
    private String studyName;
    private List<Study.Topic> topic;
    private List<Study.MeetingDays> meetingDays;
    private int count;
    private Set<String> members;
    private Set<String> wishMembers;
    private Set<String> preMembers;
}

File: Android/app/src/main/java/com/example/firstproject/data/model/dto/response/StudyDTO.kt
package com.example.firstproject.data.model.dto.response

// swagger 기준 GET/api/studies
data class StudyDTO(
    val id: String,
    val studyName: String,
    val topic: String,
    val meetingDays: List<String>,
    val count: Int,
    val members: List<String>,
    val studyContent: String,
    val createdBy:String
)